packages/migrate/src/__tests__/MigrateDev.test.ts

```diff
@@ -757,6 +752,11 @@ describe('sqlite', () => {
           └─ migration.sql
 
       Your database is now in sync with your schema.
+
+      Running seed command \`ts-node prisma/seed.ts\` ...
+
+      🌱  The seed command has been executed.
+
```

Comment:
This snapshot update describes this change well. You can notice that now seed is running after migration was applied instead of running before it.